packages/editor/src/components/post-format/index.js

```diff
@@ -57,7 +57,7 @@ function PostFormat( { onUpdatePostFormat, postFormat = 'standard', supportedFor
 					<div className="editor-post-format__suggestion">
 						{ __( 'Suggestion:' ) }{ ' ' }
 						<Button isLink onClick={ () => onUpdatePostFormat( suggestion.id ) }>
-							{ suggestion.caption }
+							{ sprintf( 'Apply the "%s" format', suggestion.caption ) }
```

Comment:
@greatislander thanks! The string needs to be translatable and use `__()`. Also, there should be a translators comment on the line immediately before the string with the placeholder.
```suggestion
							{ sprintf(
								/* translators: %s: post format */
								__( 'Apply the "%s" format.' ),
								suggestion.caption
							) }
```
It would be nice to use the same string used in `/packages/editor/src/components/post-publish-panel/maybe-post-format-panel.js` with the trailing period. The translators comment should be added there too.
I'd also consider to add some margin to the top.